stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -37,7 +34,45 @@ This module allows introducing a WebSub Subscriber Service with `onIntentVerific
  - When the service is started a subscription request is sent for a hub/topic combination, either specified as annotations or discovered based on the resource URL specified as an annotation.
  - If `onIntentVerification` is not specified, intent verification will be done automatically against the topic specified as an annotation or discovered based on the resource URL specified as an annotation.
  - If a secret is specified for the subscription, signature validation will be done for authenticated content distribution.
+ 
+**Sends the subscription request**
   
+  > When the `subscribeOnStartUp` is set to true in the Subscriber Service, it will result in a subscription request being sent to the specified hub for the specified topic, with the specified lease seconds value and the specified secret for authenticated content distribution. 
+
```

Comment:
Add following here
```
Since an `onIntentVerification` resource function is not included, intent verification for subscription and unsubscription 
requests would happen automatically, if the topic specified in the request matches that specified as an annotation or 
that discovered for the annotated resource URL.
```